Composer/packages/client/src/components/ProjectTree/ProjectTree.tsx

```diff
@@ -212,11 +213,11 @@ export const ProjectTree: React.FC<Props> = ({
 
   const renderDialogHeader = (skillId: string, dialog: DialogInfo) => {
     const warningContent = notificationMap[currentProjectId][dialog.id]
```

Comment:
is it better to put both error and warning content inside if(notificationMap\[currentProjectId\]\[dialog.id\]) instead of typescript ?

Reply:
I think it's about equivalent. I've gotten used to using `?.` for cases like this.